packages/viewer/src/IViewer.ts

```diff
@@ -88,10 +88,12 @@ export interface ViewerEventPayload {
   [ViewerEvent.LightConfigUpdated]: LightConfiguration
 }
 
-export type SpeckleView = {
-  name: string
-  id: string
-  view: { name: string; origin: Vector3Like; target: Vector3Like }
+export type SpeckleView = SpeckleObject & {
```

Comment:
@andrewwallacespeckle 
The previous `SpeckleView` structure was legacy stuff. A `SpeckleView` is just another `SpeckleObject` with extra stuff, some of which may not be filled out by connectors so I made them optional. 
For the `name` prop, there is a very good change that it will always exist, however I am not 100% sure so I put it down as optional. I've asked connectors and maybe we will get a definite answer on this. If not, I guess you could also fallback to the displaying the `id` which is mandatory to exist